modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -47,45 +66,176 @@ For more information about autoinstallation distributions and files, see xref:cl
 . Click btn:[Finish].
 . Review the details of your new profile, and customize as required.
 
-[NOTE]
-====
 When you are creating your autoinstallation profile, you can check [guimenu]``Always use the newest Tree for this base channel``.
 This setting allows {productname} to automatically pick the latest distribution that is associated with the specified base channel.
 If you add new distributions later, {productname} uses the most recently created or modified.
-====
 
-[NOTE]
-====
 Changing the Virtualization Type usually requires changes to the profile bootloader and partition options.
 This can overwrite your customization.
 Verify new or changed settings before saving them, by navigating to the [guimenu]``Partitioning`` tab.
-====
+
+The kernel options from the distribution and the profile are combined.
 
 You can change the details and settings of your autoinstallation profiles by navigating to menu:Systems[Autoinstallation > Profiles] and clicking the name of the profile you want to edit.
+Alternatively, navigate to menu:Systems[System List], select the client you want to provision, and navigate to the menu:Provisioning[Autoinstallation] subtab.
+
+
+[[autoyast]]
+== AutoYast Profiles
+
+An {ay} profile consists of a [guimenu]``Label`` that identifies the profile, an [guimenu]``Autoinstall Tree`` that points to an autoinstallation distribution, various options, and, most importantly, an {ay} installation file.
 
-To adjust advanced autoinstallation configuration options, navigate to menu:Systems[System List], select the client you want to provision, and navigate to the menu:Provisioning[Autoinstallation] subtab.
-Select the autoinstallation profile to use, and click btn:[Advanced Configuration].
-For traditional clients, use the various fields to set options for specialized {kickstart} scenarios.
-For Salt clients, provide the kernel options you require.
-For more information about these options, see https://documentation.suse.com/sles/html/SLES-all/book-sle-admin.html.
+The {ay} installation file is an XML file that give directions to the {ay} installer. {ay} calls it a "control file".
+For the full syntax of {ay} installation files, see https://doc.opensuse.org/projects/autoyast/#cha-configuration-installation-options.
 
+{suse} provides templates of {ay} installation files that you can use as a starting point for your own custom files.
+You will find them at https://github.com/SUSE/manager-build-profiles.
+Each of these profiles requires you to set some variables before you use it.
+Check the [path]``README`` file included with the script to determine which variables you need.
+For more information about using variables in {ay} scripts, see xref:client-configuration:autoinst-profiles#variables[Variables].
 
+For more information about {ay}, see https://doc.opensuse.org/projects/autoyast/.
```

Comment:
I think we should add a sub-section to AutoYaST. It should talk about the important sections in the autoyast file for installing with SUSE Manager. They are:

- `<general>$SNIPPET('spacewalk/sles_no_signature_checks')</general>` 
- `<add-on>` and how to add child channels to the installation
- `<software><products config:type="list"> ...` and how to specify the correct product for the Unified Installer
- The registration snippets for salt and traditional

Reply:
I just copied this list as a starting point.

I just don't have the time to start writing a full section, but I agree it would be worth it.

Reply:
Created new issue to add this suggestion later: https://github.com/SUSE/spacewalk/issues/14765, so this suggestion does not hold back the approval.